src/Components.tsx

```diff
@@ -91,3 +91,62 @@ export const ConnectNotificationBanner = (): JSX.Element => {
   const {wrapper} = useCreateComponent('notification-banner');
   return wrapper;
 };
+
+export const ConnectIssuingCard = ({
+  defaultCard,
+  cardArtFileLink,
+  cardSwitching,
+  fetchEphemeralKey,
+}: {
+  defaultCard?: string;
+  cardArtFileLink?: string;
+  cardSwitching?: boolean;
+  fetchEphemeralKey?: (fetchParams: {
+    nonce: string;
+    issuingCard: string;
+  }) => Promise<{
+    ephemeralKeySecret: string;
+    nonce: string;
+    issuingCard: string;
+  }>;
+}): JSX.Element => {
+  const {wrapper, component: issuingCard} = useCreateComponent('issuing-card');
+
+  React.useEffect(() => {
+    if (!issuingCard) return;
+    issuingCard.setDefaultCard(defaultCard);
```

Comment:
You could implement these today with regular setAttributes, however the setter methods right now have race conditions, so we need to wait for those to be solved first

Reply:
To clarify we need to merge https://github.com/stripe/connect-js/pull/71#pullrequestreview-1735483336 , then publish, take a dependency on this repo, then we can merge this one